operations/service.go

```diff
@@ -101,8 +101,8 @@ func startSystemCronJobs(ctx context.Context, env evergreen.Environment) {
 		units.PopulateCatchupJobs(30),
 		units.PopulateHostAlertJobs(20)))
 
-	amboy.IntervalQueueOperation(ctx, env.RemoteQueue(), 3*time.Hour, time.Now(), opts, amboy.GroupQueueOperationFactory(
-		units.CacheHistoricalTestDataJob(15)))
+	amboy.IntervalQueueOperation(ctx, env.RemoteQueue(), 6*time.Hour, time.Now(), opts, amboy.GroupQueueOperationFactory(
```

Comment:
Since our cron is imprecise, you'll want to have the interval be <6 hours so that you don't accidentally miss a run

Reply:
Got it, thanks! I'll leave it at 3 hours then.